app/src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.app.databinding
+
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import org.oppia.app.R
+import java.text.SimpleDateFormat
+import java.util.*
+
+/** Binds date text with relative time. */
+@BindingAdapter("profile:created")
+fun setProfileDataText(textView: TextView, timestamp: Long) {
+  val sdf = SimpleDateFormat("dd MMM yyyy")
```

Comment:
I think we should create one central utility file from where we should access date format or even convert date timestamp to string from that file. That way we can have consistency in date/time through the app.

Reply:
I'm going to leave this as a todo for now.